modules/serverless-create-func-kn.adoc

```diff
@@ -32,10 +28,9 @@ envs: {}
 +
 [source,terminal]
 ----
-$ kn func create <path> -r <registry> -l <runtime> -t <trigger> -i <image> -n <namespace>
+$ kn func create <path> -l <runtime> -t <template>
```

Comment:
Would <template_name> or <template_ID> be clearer here?

Reply:
good point, this made me actually notice a few things in the changes that I have questions about, thanks Max

Reply:
I think it's OK to leave this as template, since it's a type of template you're specifying, similar to `runtime`